jobs/src/main/scala/dcos/metronome/model/CronSpec.scala

```diff
@@ -70,3 +77,44 @@ object CronSpec {
     }
   }
 }
+
+object CronSpecValidation {
+  val validDayOfMonth = "Day of the month must exist in the provided month and year (e.g. February have only < 29 days so running cron on 30.2. is invalid)"
```

Comment:
several nits here:
1. the message is incorrect... it doesn't have anything to do with the year... This is valid day of the month or within the possible days of a given month.
2. Feb has < 30... or =< 29.   29 is a valid day of feb in leap years
3. I don't like the notation of 30.2.   we don't need to "americanize" it... but it is made more clear to write Feb 30 IMO.   There isn't room for interpretation.